package.json

```
@@ -29,7 +29,7 @@
     "description": "SEO Data Transporter helps you transfer post/page specific SEO data, like custom doctitles, custom META descriptions and keywords, etc., from one platform (theme or plugin) to another.",
     "author": "StudioPress",
     "authoruri": "http://www.studiopress.com/",
```

@ethan-schaffer would you mind updating this adding the `s` here?

```suggestion
    "authoruri": "https://www.studiopress.com/",
```